cmd/internal/root.go

```diff
@@ -60,23 +60,29 @@ var rootCmd = &cobra.Command{
 		log.Infof("Starting cf-operator %s with namespace %s", version.Version, cfOperatorNamespace)
 		log.Infof("cf-operator docker image: %s", converter.GetOperatorDockerImage())
 
-		host := viper.GetString("operator-webhook-service-host")
-		port := viper.GetInt32("operator-webhook-service-port")
+		operatorWebhookHost := viper.GetString("operator-webhook-service-host")
```

Comment:
I replaced this with the short names, because introducing another very specific variable name was confusing to readers. (operator webhook host with operator webhook service host vs webhook server host ....)

Also the scope is really short lived, so a short variable name is fine.